File: weisi/src/main/java/com/cjq/lib/weisi/protocol/OnSensorInfoAnalyzeListener.java
package com.cjq.lib.weisi.protocol;

/**
 * Created by CJQ on 2018/1/11.
 */

public interface OnSensorInfoAnalyzeListener {
    void onSensorInfoAnalyzed(int sensorAddress,
                              byte dataTypeValue,
                              int dataTypeIndex,
                              long timestamp,
                              float batteryVoltage,
                              double rawValue);
}

File: wsb/src/main/java/com/wsn/lib/wsb/protocol/OnSensorInfoAnalyzeListener.kt
package com.wsn.lib.wsb.protocol

/**
 * Created by CJQ on 2018/1/11.
 */

interface OnSensorInfoAnalyzeListener {
    fun onSensorInfoAnalyzed(sensorAddress: Int,
                             dataTypeValue: Byte,
                             dataTypeIndex: Int,
                             timestamp: Long,
                             batteryVoltage: Float,
                             rawValue: Double)
}
